src/constants/global.constants.ts

```diff
@@ -30,3 +30,4 @@ export const allowedManualRunExtensions = ["python", "starlark"];
 export const AKRoutes = isProduction ? Sentry.withSentryReactRouterV7Routing(Routes) : Routes;
 export const sentryDsn = import.meta.env.SENTRY_DSN;
 export const maxLogsPageSize = 100;
+export const searchByTermDebounceTime = 1000;
```

Comment:
I'm not sure it's a global constant; it's more like a components subdirectory or a components.constants.ts file.
I know that previously I insisted the constants would be in the constants directory, but these days I think that if we're talking about 1 or 2 consts which are related to a specific component, we can leave it in the component's file.

What do you think?